app/templates/file_set_metadata.html.haml

```diff
@@ -85,8 +85,10 @@
         %span{:property => "http://schema.org/url"}
           &= package.related_websites.pluck(:url).join(", ")
         %h1 Contributors
-        %span{:property => "http://schema.org/name"}
-          &= package.contributors.pluck(:name).join(", ")
+        - package.contributors.each do |contributor|
+          %span{:property => "http://schema.org/name"}
+            &= contributor.name
+          %br/
```

Comment:
@Tao-Liu - I'm sure it doesn't matter but there is an unnecessary trailing / on this line.